Dockerfile

```
@@ -16,19 +16,18 @@ RUN mkdir -p /etc/apt/keyrings && \
         https://deb.nodesource.com/node_18.x nodistro main" | \
     tee /etc/apt/sources.list.d/nodesource.list && \
     apt-get update && \
-    apt-get install nodejs -y
+    apt-get install nodejs -y &&\
+    apt-get clean
 
-# Install standard R so that we can use PPM binaries
-RUN apt-get install -y --no-install-recommends software-properties-common && \
-    add-apt-repository --enable-source --yes "ppa:marutter/rrutter4.0" && \
-    add-apt-repository --enable-source --yes "ppa:c2d4u.team/c2d4u4.0+" && \
-    apt-get purge -y software-properties-common && \
-    apt-get autoremove -y
+RUN curl -L https://rig.r-pkg.org/deb/rig.gpg -o /etc/apt/trusted.gpg.d/rig.gpg &&\
+    sh -c 'echo "deb http://rig.r-pkg.org/deb rig main" > /etc/apt/sources.list.d/rig.list' &&\
+    apt update && \
+    apt install r-rig && \
+    apt-get clean
```

IIRC we should use `apt-get` rather than `apt` in scripts.

```suggestion
    apt-get update && apt-get install r-rig -y
```

The `-y` is not strictly required here but if `r-rig` adds dependencies later it means they'll be installed too.